package.json

```diff
@@ -1,10 +1,17 @@
 {
   "name": "artsy-morgan",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "main": "index.js",
   "license": "MIT",
+  "scripts": {
+    "test": "NODE_ENV=staging mocha -r should"
```

Comment:
I believe we can set `NODE_ENV=test` during tests, as it only short-circuits when `NODE_ENV === 'development'`